src/test/kotlin/org/ionproject/integration/step/tasklet/iseltimetable/implementations/DownloadAndCompareTaskletTest.kt

```diff
@@ -0,0 +1,133 @@
+package org.ionproject.integration.step.tasklet.iseltimetable.implementations
+
+import java.io.File
+import java.nio.file.Path
+import org.ionproject.integration.IOnIntegrationApplication
+import org.ionproject.integration.file.exceptions.InvalidFormatException
+import org.ionproject.integration.file.exceptions.ServerErrorException
+import org.ionproject.integration.job.ISELTimetable
+import org.ionproject.integration.step.utils.SpringBatchTestUtils
+import org.ionproject.integration.utils.Try
+import org.ionproject.integration.utils.orThrow
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertNull
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.Mockito
+import org.springframework.batch.core.StepContribution
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.autoconfigure.batch.BatchAutoConfiguration
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.TestPropertySource
+import org.springframework.test.context.junit.jupiter.SpringExtension
+
+@ExtendWith(SpringExtension::class)
+@ContextConfiguration(classes = [ISELTimetable::class, DownloadAndCompareTasklet::class, BatchAutoConfiguration::class, IOnIntegrationApplication::class])
+@TestPropertySource(properties = ["local-file-destination=src/test/resources/TIMETABLE-SUCCESSFUL.pdf", "pdf-remote-location:https://www.isel.pt/media/uploads/LEIC_0310.pdf"])
```

Comment:
Isn't this line too long? Namely, does it pass style checking?

Reply:
Broke it down to several lines.